### PR TITLE
Fixed issue #6558

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#6558

## Description
Fixed CSS issue on booking product

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Nor working, please find the screenshot [Here](https://prnt.sc/BbV4hMBqml6x)
**please compile the file**  run **_npm run prod_**

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
This is  working fine, please find the screenshot [Here](https://prnt.sc/Qb6IeU-5q7Qq)

File: packages/Webkul/BookingProduct/src/Resources/assets/sass/velocity.scss [hunk 1]
```diff
@@ -258,4 +269,13 @@
...
+}
+
+body {
+        &.rtl {
+        .book-cross-icon {
+            left: 80px !important;
+            right: inherit !important;
+        }
+    }
 }
```
Comment (jitendra-webkul) [inline packages/Webkul/BookingProduct/src/Resources/assets/sass/velocity.scss:281]: body {
        &.rtl {
        .booking-information {
            .cross-icon {
                left: 80px !important;
                right: inherit !important;
            }
       }
    }
}

**Comment by akathuria289** [comment]
Done

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**